docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+services:
```

Comment:
Didn't this `docker-compose` file already exist somewhere in the repo? Any reason we needed to copy it here again? I could be wrong though.